docs/tutorials/quickstart.md

```diff
@@ -0,0 +1,237 @@
+# Get started with Coder
+
+<div style="padding: 0px; margin: 0px;">
+  <span style="vertical-align:middle;">Author: </span>
+  <a href="https://github.com/EdwardAngert" style="text-decoration: none; color: inherit; margin-bottom: 0px;">
+    <span style="vertical-align:middle;">Edward Angert</span>
+    <img src="https://avatars.githubusercontent.com/u/17991901?v=4" width="24px" height="24px" style="vertical-align:middle; margin: 0px;"/>
+  </a>
+</div>
+September 23, 2024
+
+---
+
+Follow the steps in this guide to install Coder locally or on a cloud-hosting
+provider, set up a workspace, and connect to it from VS Code.
+
+By the end of this guide, you'll have a remote development environment that you
+can connect to from any device anywhere, so you can work on the same files in a
+persistent environment from your main device, a tablet, or your phone.
+
+## Install and start Coder
+
+<div class="tabs">
+
+## Linux/macOS
+
+1. Install Docker:
+
+   ```bash
+   curl -sSL https://get.docker.com | sh
+   ```
+
+   For more details, visit:
+
+   - [Linux instructions](https://docs.docker.com/desktop/install/linux-install/)
+   - [Mac instructions](https://docs.docker.com/desktop/install/mac-install/)
+
+1. Assign your user to the Docker group:
+
+   ```shell
+   sudo usermod -aG docker $USER
+   ```
+
+1. Update the group with `newgrp`:
```

Comment:
```suggestion
1. Log out and log back in so that your group membership is re-evaluated or run:
```
as per https://docs.docker.com/engine/install/linux-postinstall/